go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	code.cloudfoundry.org/lager/v3 v3.0.3
 	code.cloudfoundry.org/localip v0.0.0-20230406154046-f137f65d303d
 	code.cloudfoundry.org/locket v0.0.0-20230406154009-5e8522d975d2
-	code.cloudfoundry.org/runtimeschema v0.0.0-20231212222654-3a68a52a50c2
+	code.cloudfoundry.org/runtimeschema v0.0.0-20231214123546-c95bb31afd3e
```

Comment:
note that we'll have to make sure that commit stays correct as that is the commit on the [branch](https://github.com/cloudfoundry/runtimeschema/tree/readiness-hc-event) from which a [pr](https://github.com/cloudfoundry/runtimeschema/pull/19) was made.  Not the master branch in runtimeschema.  Reasons it could disappear is branch is deleted and the commit is squashed and merged rather than merged (squash and merge changes the commit sha)